gettsim/tests/test_data/grundrente_proxy_rente.csv

```diff
@@ -1,9 +1,9 @@
 "p_id","tu_id","hh_id","jahr","entgeltp","wohnort_ost","priv_rente_m","jahr_renteneintr","alter","bruttolohn_m","geburtsjahr","rentner","rente_vorj_vor_grundr_proxy_m","note"
-1,1,1,2021,20,"False","0","2016",70,0,1951,"True","675.59","made up, regression test"
-2,2,2,2021,30,"False","0","2016",70,0,1951,"True","1013.39","made up, regression test"
-3,3,3,2021,20,"True","0","2016",70,0,1951,"True","656.62","made up, regression test"
-4,4,4,2021,20,"False","1000","2016",70,0,1951,"True","1675.59","made up, regression test"
-5,5,5,2021,20,"False","0","2014",70,0,1951,"True","626.36","made up, regression test"
-6,6,6,2021,20,"False","0","2020",70,0,1951,"True","834.24","made up, regression test"
-7,7,7,2021,20,"False","0","1996",90,0,1931,"True","683.8","made up, regression test"
+1,1,1,2021,20,"False","0","2016",70,0,1951,"True","673.54","made up, regression test"
```

Comment:
Please add test cases for the edge cases, e.g., year 2000 and age 63. I do not see through how this is interacting with `regelaltersgrenze`.

Reply:
The change in regelaltersgrenze affects the deductions of benefits, therefore the made-up pension payments (by the previous gettsim implementation) used in these test cases. In the fixed statutory retirement age is 1 month higher now for the cases , therefore penalties are higher and pensions lower. But yes, it seems like we need better tests for the implementation of the regular pension payments.